Gemfile

```diff
@@ -1,7 +1,8 @@
 source "https://rubygems.org"
 
-gem 'github-pages', group: :jekyll_plugins
-gem 'jekyll-feed', group: :jekyll_plugins
-gem 'jekyll-redirect-from', group: :jekyll_plugins
-gem 'jekyll-sitemap', group: :jekyll_plugins
-
+group :jekyll_plugins do
+  gem 'github-pages'
+  gem 'jekyll-feed'
```

Comment:
Do we need that `jekyll-feed` gem?

Reply:
Good question, I'm not sure. This PR doesn't add or remove any gems, its updating the ones that are here.

Reply:
Yea totally. Is there any issue with merging this PR as is?

We can move the dependency improvements into another PR.

Reply:
No probs. It builds on my machine, so LGTM :smile: